### DISCUSSION TITLE
WordPress 6.3- Editeur en disfonctionnement

### DISCUSSION THREAD

**Discussion by Comprendre-IT** [community/community#64609] [Other Feature Feedback, Questions, & Ideas]
Bonjour,

J'utilise WordPress 6.3 en local grâce à mon application local version 7.1. J'ai créé un site web avec les paramètres par défaut. 
Je ne peux pas modifier ou créer un article, à chaque fois que je veux "sauvegarder" j'ai comme un message d'erreur qui apparait.

![WordPress_Editeur- erreur inattendue](https://github.com/community/community/assets/141761707/1f8970be-aed2-41cd-980c-cb1bb37f5daa)

Si vous pouvez m'orienter vers un blog, article, conseils et astuces qui peut m'aider, car j'ai constaté que ce problème est très connu il apparait sur plusieurs forums depuis 2021. J'ai déjà sollicité WPFR, WP engine, Flywheel, les sujets déposés restent sans solutions.

En vous remerciant d'avance, merci.

**Comment by queenofcorgis** [comment]
Hi @Comprendre-IT ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Wordpress. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
